tools/clang/include/clang/SPIRV/Type.h

```diff
@@ -68,39 +71,51 @@ class Type {
   static const Type *getFloat16(SPIRVContext &ctx);
   static const Type *getFloat32(SPIRVContext &ctx);
   static const Type *getFloat64(SPIRVContext &ctx);
+
   static const Type *getVector(SPIRVContext &ctx, uint32_t component_type,
```

Comment:
It seems we can kill `getVector`.  These types are only serving the purpose of SPIR-V codegen; we will surely only generate vec2, vec3, or vec4.